sycl/test-e2e/no-xfail-without-tracker.cpp

```diff
@@ -0,0 +1,44 @@
+// This test is intended to ensure that we have no trackers marked as XFAIL
+// without a tracker information added to a test.
+//
+// The format we check is:
+// XFAIL: lit,features
+// XFAIL-TRACKER: intel/llvm issue URL|intel/llvm#NNNNN shortcut
+//
+// REQUIRES: linux
+//
+// Explanation of the command:
+// - search for all "XFAIL" occurences, display line with match and the next one
+//   -I, --include to drop binary files and other unrelated files
+// - in the result, serach for "XFAIL" again, but invert the result - this
+//   allows us to get the line *after* XFAIL
+// - in those lines, check that XFAIL-TRACKER is present and correct. Once
+//   again, invert the search to get all "bad" lines
+// - make a final count of how many ill-formatted directives there are and
+//   verify that against the reference
+//
+// RUN: grep -rI "XFAIL:" %S -A 1 --include=*.c --include=*.cpp \
+// RUN:     --no-group-separator | \
+// RUN: grep -v "XFAIL:" | \
+// RUN: grep -Pv "XFAIL-TRACKER:\s+.*intel/llvm.*\d+" | \
+// RUN: wc -l | FileCheck %s --check-prefix NUMBER-OF-XFAIL-WITHOUT-TRACKER
+//
+// The number below is a number of tests which are *impropertly* XFAIL-ed, i.e.
+// we either don't have a tracker associated with a failure listed in those
+// tests, or it is listed in a wrong format.
+// Note: strictly speaking, that is not amount of files, but amount of XFAIL
+// directives. If a test contains several XFAIL directives, some of them may be
+// valid and other may not.
+//
+// That number *must not* increase. Any PR which causes this number to group
```

Comment:
```suggestion
// That number *must not* increase. Any PR which causes this number to group
```

Is `group` a typo? I guess you meant `grow`?

Reply:
Right, I type `group` way more often than `grow`, so my hands betray me here :)